stages/io.weldr.noop

```diff
@@ -3,7 +3,7 @@
 import json
 import sys
 
-def main(tree, options):
+def main(_tree, options):
```

Comment:
Is this because it's an unused option? Seems weird to me.

Reply:
Assuming that's what it is: this is the same logic as Rust uses, and I must say I quite like it. You have to be explicit about unused variables.

Reply:
Yes, it is because of the unused option. I wanted to preserve it as we use it in every stage.